apps/email/lib/email_web/templates/shared/pre.html.eex

```diff
@@ -0,0 +1,3 @@
+<pre style="font-size:18px;line-height:24px;text-align:left">
```

Comment:
```suggestion
<pre style="font-size:14px;line-height:24px;text-align:left">
```

18px is enormous with fixed-width text. Here's how it would look at 14px
<img width="578" alt="Screenshot 2023-08-29 at 10 43 09 AM" src="https://github.com/pluralsh/plural/assets/85062/a7d82e71-e235-4bc5-a497-e4f11d45537f">